### PR TITLE
Translate Zen to English

### PR DESCRIPTION
**Pull request by mauricioabreu**
English is the language to go worldwide.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Zen of Codebakers
 -----------------
 
-Zen of Codebakers é um pacote Python que, com palavras, demonstra a ideologia de um time de engenheiros de software que buscam constante evolução.
+Zen of Codebakers is a Python package that is here to show the ideology and mindset of a software engineering team pursuing constant evolution.
```
Comment (gtonello) [inline README.rst]: Uhm, looks weird to me ".. that is here to show.."
Maybe without *here* sounds better. What do you think?

File: codebakers/__init__.py [hunk 2]
```diff
@@ -3,25 +3,25 @@
...
-Não deixa a palavra 'mantenedor' te deixar preguiçoso quando você não o é.
-Codifique de uma maneira fácil de entender.
-Não utilize propriedades obscuras das ferramentas para resolver problemas.
+Empathy wins over code and technical decisions.
+Remember the following Kent Beck rules:
+    Passes the tests
+    Reveals intention
+    No duplication
+    Fewest elements
+Look for a easy deploy in order to deliver faster and better.
```
Comment (gtonello) [inline codebakers/__init__.py]: ..looks for *an* easy deploy..

File: codebakers/__init__.py [hunk 3]
```diff
@@ -3,25 +3,25 @@
...
+The smaller delivery, the better result is achieved.
+Choose the easier option to change later.
+Code without tests does not exist. Code with bad tests neither.
+Seek to implement relevant tests. Do not trust blind coverage.
+Thirdy party component should have integration tests.
+Don't take code review as a personal criticism.
+Avoid callbacks when it is not the better solution.
+Document code whenever you judge it complex.
+Don't let the 'maintainer' word give you laziness.
+Code in a easy way to understand.
```
Comment (gtonello) [inline codebakers/__init__.py]: .. in an easy way..

**Comment by gtonello** [comment]
:+1: perfect!

File: codebakers/__init__.py [hunk 4]
```diff
@@ -3,25 +3,25 @@
...
-Codifique de uma maneira fácil de entender.
-Não utilize propriedades obscuras das ferramentas para resolver problemas.
+Empathy wins over code and technical decisions.
+Remember the following Kent Beck rules:
+    Passes the tests
+    Reveals intention
+    No duplication
+    Fewest elements
+Look for a easy deploy in order to deliver faster and better.
+If deploying code turns into a problem, increasing deliver is not a solution.
```
Comment (rzanluchi) [inline codebakers/__init__.py]: I think you changed the meaning of this phrase

File: codebakers/__init__.py [hunk 5]
```diff
@@ -3,25 +3,25 @@
...
+The smaller delivery, the better result is achieved.
```
Comment (rzanluchi) [inline codebakers/__init__.py]: a better result is achieved

File: README.rst (hunk 1 above)
Comment (mauricioabreu) [inline README.rst]: :+1:

File: codebakers/__init__.py (hunk 2 above)
Comment (mauricioabreu) [inline codebakers/__init__.py]: :+1:

File: codebakers/__init__.py (hunk 5 above)
Comment (mauricioabreu) [inline codebakers/__init__.py]: In english "quanto menor a/o <substantivo>" is translated to "the smaller, the better, the worse" etc.

Comment (rzanluchi) [inline codebakers/__init__.py]: :+1:

File: codebakers/__init__.py (hunk 4 above)
Comment (mauricioabreu) [inline codebakers/__init__.py]: How could it be better? Any idea?

Comment (rzanluchi) [inline codebakers/__init__.py]: oh no, just read it wrong :)

**Comment by victorpoluceno** [comment]
:+1:

**Pull request merged**